Framework/LiveData/inc/MantidLiveData/ISIS/ISISHistoDataListener.h

```diff
@@ -131,6 +129,10 @@ class ISISHistoDataListener : public API::LiveListener {
   /// Time regime to load
   int m_timeRegime;
 
+  /// Buffer workspace to store instrument data (or not only instrument in the
+  /// future), prevents loading for every chank of data
+  API::MatrixWorkspace_sptr bufferWorkspace;
```

Comment:
Minor point: Mantid coding standards has prefix `m_` for member variables see http://developer.mantidproject.org/Standards/CPPStandards.html#naming-conventions

Reply:
Done